block-storage-attacher/images/iscsi-attach.sh

```diff
@@ -7,6 +7,7 @@ LOG=/var/log/ibmc-block-attacher-service.log
 INITIATOR=/etc/iscsi/initiatorname.iscsi
 ISCSI_CONF=/etc/iscsi/iscsid.conf
 #ISCSIADM=/sbin/iscsiadm
+iscsi_restart=true
```

Comment:
1. can't we make ..by default as "iscsi_restart=false"  ?  and then do checks to see if creds changed and make it "true"
2. we should break from other checks..once we discover "iscsi_restart" is required.